RecoLocalCalo/HcalRecAlgos/interface/HcalDeterministicFit.h

```diff
@@ -58,6 +60,22 @@ class HcalDeterministicFit {
   0.0224483, 0.0210872, 0.0197684, 0.0184899, 0.01725, 0.0160471, 0.0148795, 0.0137457, 0.0126445, 
   0.0115743, 0.0105341, 0.00952249, 0.00853844, 0.00758086, 0.00664871,0.00574103, 0.00485689, 0.00399541, 
   0.00315576, 0.00233713, 0.00153878, 0.000759962, 0 };
+
+  static constexpr float siPM205Frac[] = {
```

Comment:
I was actually asking for an std::array, not just the explicit size.
The current update is acceptable.